test cases/common/59 object generator/meson.build

```diff
@@ -1,6 +1,9 @@
 project('object generator', 'c')
 
-python = find_program('python3')
+python = find_program('python3', required : false)
+if not python.found()
+  python = find_program('python')
+endif
```

Comment:
Should probably use the python3 module here since we don't use `find_program` on a script.